packages/web/src/pages/rewards-page/Tiers.tsx

```diff
@@ -282,40 +283,59 @@ const TierColumn = ({
       />
       {(
         Object.keys(messages.features) as Array<keyof typeof messages.features>
-      ).map((feature) => (
-        <Flex key={feature} pv='m' borderTop='default' justifyContent='center'>
-          <Text>
-            {tierFeatures[feature] ? (
-              <Flex h={24} direction='row' alignItems='center' gap='m'>
-                <IconValidationCheck />
-                {feature === 'customDiscordRole' && current ? (
-                  <Tooltip text='Refresh Discord role'>
-                    <Button
-                      size='small'
-                      variant='secondary'
-                      iconLeft={IconRefresh}
-                      onClick={onClickDiscord}
-                    />
-                  </Tooltip>
-                ) : null}
-              </Flex>
-            ) : (
-              <Flex h={24} w={24} alignItems='center' justifyContent='center'>
-                <Flex
-                  h={16}
-                  w={16}
-                  borderRadius='circle'
-                  border='strong'
-                  css={{
-                    borderWidth: 2,
-                    borderColor: 'var(--harmony-neutral)'
-                  }}
-                />
-              </Flex>
-            )}
-          </Text>
-        </Flex>
-      ))}
+      ).map((feature) => {
+        const minAudio =
+          badgeTiers.find((b) => b.tier === tier)?.minAudio.toString() ?? '0'
+
+        return (
+          <Flex
+            key={feature}
+            pv='m'
+            borderTop='default'
+            justifyContent='center'
+          >
+            <Text>
+              {feature === 'balance' ? (
+                <Flex h={24} alignItems='center' justifyContent='center'>
```

Comment:
nit: im a fan of the `unit<n>` spacing variables, if we want to make things more compact or spaced out in the future we can update those and the app will grow as one

Reply:
i didn't think these h values supported the unit<n> stuff